### PR TITLE
RIA-7332 Document for Appellant ftp decided - refused

### PR DESCRIPTION
**Pull request by Ryan-Nesbitt**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/RIA-7332

### Change description ###

Document generation for Appellant FTPA decided - refused.
ADA - 7 working days
Non-ADA - 14 calendar days

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/functionalTest/resources/scenarios/RIA-7332-internal-appellant-ftpa-decided-refused-document.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+              "suppliedBy": "",
+              "description": "",
+              "dateUploaded": "{$TODAY}"
+            }
+          }
+        ]
+      }
+    }
+  }
+}
```
Comment (hussain-faruk) [inline src/functionalTest/resources/scenarios/RIA-7332-internal-appellant-ftpa-decided-refused-document.json:48]: I think we should have 2 FTs per scenario (granted/partially granted/refused), one for ADA and one for non-ada. Although they both should work as the handler conditions have just isAppellantInDetention, having 2 FTs ensures against future changes.

**Review by hussain-faruk — CHANGES_REQUESTED** [review]
FT for both ada and non-ada required

File: src/functionalTest/resources/scenarios/RIA-7332-internal-appellant-ftpa-decided-refused-document.json (hunk 1 above)
Comment (Ryan-Nesbitt) [reply src/functionalTest/resources/scenarios/RIA-7332-internal-appellant-ftpa-decided-refused-document.json]: Added

**Review by hussain-faruk — APPROVED** [review]

**Review by colmlynch60 — APPROVED** [review]
LGTM

**Pull request merged**